### DISCUSSION TITLE
Simple benchmark comparison with Flexsearch

### DISCUSSION THREAD

**Discussion by paulocoghi** [jameslittle230/stork#113] [General]
Hello, James! First of all, congratulations on Stork!

I would like to suggest to add Stork on the Flexsearch benchmarks (repo [here](https://github.com/nextapps-de/flexsearch), benchs [here](https://github.com/nextapps-de/flexsearch/tree/master/test))

I understand that Stork doesn't present itself as the fastest, but just as a fast option. Thus, including it on the Flexsearch benchmark, which presents itself as the fastest _(and currently it seems to be)_ it would be interesting to discover:

1. How fast Stork is compared to the fastest
2. How much it can improve, in case it isn't as fast as Flexsearch, learning from the later approach.
3. Since Stork seems to be the first to use WebAssembly to accelerate things, imagine how fast a new approach could be, where the best of Flexsearch and Stork play together :)

Thanks!

**Comment by jameslittle230** [comment]
Thanks for sharing! I haven't put too much time into benchmarking, but this seems like a good place to look when I get started!
